tst/testspecial/broken-test.g.out

```diff
@@ -0,0 +1,50 @@
+gap> Test("broken-test-1.tst");
+Error, Cannot put a '> ' continuation in the middle of test output at line 3 of test file at GAPROOT/lib/test.gi:24 called from
+testError( "Cannot put a '> ' continuation in the middle of test output" 
+ ); at GAPROOT/lib/test.gi:114 called from
+ParseTestInput( full, opts.ignoreComments 
+ ) at GAPROOT/lib/test.gi:558 called from
+<function "Test">( <arguments> )
+ called from read-eval loop at *stdin*:2
+type 'quit;' to quit to outer loop
+brk> quit;
+gap> Test("broken-test-2.tst");
+Error, #@ commands cannot occur in the middle of a single test at line 5 of test file at GAPROOT/lib/test.gi:24 called from
+testError( "#@ commands cannot occur in the middle of a single test" 
+ ); at GAPROOT/lib/test.gi:124 called from
+ParseTestInput( full, opts.ignoreComments 
+ ) at GAPROOT/lib/test.gi:558 called from
+<function "Test">( <arguments> )
+ called from read-eval loop at *stdin*:2
+type 'quit;' to quit to outer loop
+brk> quit;
+gap> Test("broken-test-3.tst");
+Error, Invalid data at line 5 of test file at GAPROOT/lib/test.gi:24 called from
+testError( "Invalid data" ); at GAPROOT/lib/test.gi:130 called from
```

Comment:
Apparently this line causes a diff; perhaps you need to run the regenerate script once again?

Reply:
Something weird is going on, I am getting the test working on my machine. I tried making the line length longer and that didn't fix it either... investigating

Reply:
I have no idea why this isn't working. I checked the screen width isn't getting set too small. Any suggestions welcome.